tests/func/test_api.py

```diff
@@ -133,6 +138,7 @@ def test_missing(repo_dir, dvc_repo, remote_url):
 
     # Remove cache to make foo missing
     shutil.rmtree(dvc_repo.cache.local.cache_dir)
+    os.remove(repo_dir.FOO)
```

Comment:
Why is this one needed? Because it will trigger https://github.com/iterative/dvc/pull/2566/files#diff-16d20341660af559adf7d9e7306c760dR458 ?

Reply:
yes